chia-blockchain/templates/monit_notify.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Webhook URL where the alert will be sent
+WEBHOOK_URL="https://alert-receiver.chiaops.com/ips-info"
+HOSTNAME=$(/bin/hostname)
+BEARER_TOKEN="{{ CHIA_ALERTS_RECEIVER_TOKEN | quote }}"
+
+# Get the Tailscale IPv4 address
+TAILSCALE_IPV4=$(tailscale ip | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}' | head -n 1)
```

Comment:
We have tailscale IPs internally, but not always externally. I think we can omit this part, and just send the hostname in the alert

Reply:
Removed tailscale IPs